uefi-mkconfig

```diff
@@ -147,18 +148,20 @@ backup_entry () {
 
 main () {
 	efi_parttype="c12a7328-f81f-11d2-ba4b-00a0c93ec93b"
-	mounted_efi_partitions=$(lsblk -lo NAME,MOUNTPOINTS,PARTTYPE | grep  "$efi_parttype" | grep "/" | cut -d' ' -f1 | sort -r)
+	#as some BIOSes ignore boot order, have to force the ascending one
+	mounted_efi_partitions=$(lsblk -lo NAME,MOUNTPOINTS,PARTTYPE | grep  "$efi_parttype" | grep "/" | cut -d' ' -f1 | sort )
 	proc_kernel_commands=
 	valid_kernel_commands=
 	kernel_prefixes="vmlinuz vmlinux- kernel- bzImage zImage"
-	
+        created_IDs=()
```

Comment:
This is not used anywhere I think

Reply:
yes, this variable is added but not used